homeassistant/components/playstation_network/media_player.py

```diff
@@ -0,0 +1,107 @@
+"""Media player entity for the Playstation Network Integration."""
+
+import logging
+
+from homeassistant.components.media_player import (
+    MediaPlayerDeviceClass,
+    MediaPlayerEntity,
+    MediaPlayerEntityDescription,
+    MediaPlayerState,
+    MediaType,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import PlaystationNetworkCoordinator
+from .entity import PlaystationNetworkEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Media Player Entity Setup."""
+    coordinator: PlaystationNetworkCoordinator = config_entry.runtime_data
+
+    if coordinator.data.platform is None:
+        username = coordinator.data.username
+        _LOGGER.warning(
+            "No console found associated with account: %s. -- Skipping creation of media player",
+            username,
+        )
+        return
+
```

Comment:
Instead of skipping, make a callback function and create the devices at runtime

Reply:
Do you have an example of this?

Reply:
I've done this in Habitica button entity and in the Bring integration but that is not merged yet https://github.com/home-assistant/core/blob/0c1752baa7efe4456aa1ab20f4b964cb2cee585c/homeassistant/components/bring/todo.py